src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java

```diff
@@ -119,26 +90,13 @@ public abstract class DiseaseAnnotation extends Association {
 	@JoinTable(indexes = { @Index(columnList = "diseaseannotation_id"), @Index(columnList = "evidencecodes_curie")})
 	private List<ECOTerm> evidenceCodes;
```

Comment:
Would evidences move up also i am assuming this is precursor to pheno annotations?

Reply:
Ahh ok